backend/tests/test_check_tdd_red.py

```
@@ -0,0 +1,552 @@
+    assert len(offences) == 2
+    files_seen = {o.file for o in offences}
+    assert files_seen == {f1, f2}
+
+
+# ---------------------------------------------------------------------------
+# AC-5.3 (edge cases)
+# ---------------------------------------------------------------------------
+
+
+@pytest.mark.unit
+def test_removed_marker_does_not_flag(tmp_path: Path, monkeypatch) -> None:
```

Ruff (0.15.13)

[warning] Unused function argument: `monkeypatch`

(ARG001)